data/migrations/20200410153301_create_users_table.js

```diff
@@ -0,0 +1,7 @@
+exports.up = function(knex) {
+    
```

Comment:
I'm not really sure, why these function bodies are blank. Are they place holder? Are they simply there to ensure that tests can run? 

If so, I'd recommend adding comments to the function bodies themselves so that future me knows what is going here.